internal/env/constants.go

```diff
@@ -29,4 +29,12 @@ const (
 
 	// EnvDebug describes the internal environment variable for more verbose logging.
 	EnvDebug = `DEBUG`
+
+	// EnvPicker describes the environment variable for fuzzy support, It can value
+	// fzf or sk. If this is not set then fzf is taken as default picker.
```

Comment:
> If this is not set then fzf is taken as default picker.

My expectation of this feature was that people can set this to anything and we can feature-proof the tool. e.g. imagine a tool named `abc` is developed in the future, I should be able to say `PICKER=abc --opt1 -opt2`.

I think this is how more common env vars like EDITOR or PAGER in posix systems work. The caller has no information about the program it's calling other than it knows how to pass it the information (either via xargs or via stdin).

Reply:
Also this way, we can retain fzf logic and have an override via `PICKER`. The source code does not need to know about `sk` at all.